freqtrade/freqtradebot.py

```diff
@@ -1048,7 +1048,7 @@ def _notify_sell(self, trade: Trade, order_type: str) -> None:
             'open_rate': trade.open_rate,
             'current_rate': current_rate,
             'profit_amount': profit_trade,
-            'profit_percent': profit_percent,
+            'profit_ratio': profit_ratio,
```

Comment:
If we change the dict key here - we'll also have to change it in `webhook-config.md` - otherwise we document one thing, but it's no longer true ... and this is something we'll NEVER find unless someone runs into it :(

Reply:
adjusted the tests and the docs to use `profit_ratio` name of the item instead of `profit_percent`.

The real problem with webhook is that it uses `.format` instead of f-string formatting, so I cannot  write "profit: {profit_amount:8f} {stake_currency} ({profit_ratio * 100}%)" in webhooksell, for example...

Reply:
As a workaround, we could insert both `profit_percent` and `profit_ratio` into the dict of values, prepared by `notify_sell()` in freqtradebot

Reply:
you can't use f-strings for webhooks - at least i would not know how - as the f-string itself needs to be available beforehand.
i've never seen any example where the f-string itself comes from a variable - and while searching for that a while ago i found that that won't work.

Reply:
It's possible, without eval:
https://stackoverflow.com/a/57597617

Reply:
well eval ... but using `eval()` on user-provided strings makes the hair on my neck stand (and run far far far away) ...

Reply:
> well eval ... but using `eval()` on user-provided strings makes the hair on my neck stand (and run far far far away) ...

agree, so just FYI, achievable if evals are allowed